interfaces/Portalicious/src/app/components/form/form-sidebar.component.ts

```diff
@@ -34,6 +36,7 @@ export class FormSidebarComponent<
   readonly visible = model<boolean>(false);
   readonly formTitle = input.required<string>();
   readonly modal = model<boolean>(true);
+  readonly sideBarForm = viewChild<ElementRef<HTMLFormElement>>('sideBarForm');
```

Comment:
```suggestion
  readonly sideBarForm = viewChild.required<ElementRef<HTMLFormElement>>('sideBarForm');
```

Reply:
I was doubting between this and "only the question-mark" below. ;)
And that won out because its only 1 character. :P